nci_seer/web_client/templates/itemView.pug

```diff
@@ -8,7 +8,7 @@
     finished: ['quarantine']
   };
   var buttons = {
-    process: {name: 'Process', title: 'Keep a copy of the original image, redact marked fields, and move the result to the Audited location'},
+    process: {name: 'Process', title: 'Keep a copy of the original image, redact marked fields, and move the result to the processed folder'},
     reject: {name: 'Reject', title: 'Select this if redaction is insufficient.  The image will be moved to the rejected folder'},
     quarantine: {name: 'Quarantine', title: 'Move this image to the quarantine folder for redaction'},
     unquarantine: {name: 'Undo Quarantine', title: 'Move this image back to its location before quarantining'},
```

Comment:
where does `back to its location` mean?

Reply:
You can quarantine an item from anywhere in the system except the quarantine folder.  This puts it back to its location before quarantining (it undoes the quarantine action).

Reply:
Any suggestions on clearer wording?

Reply:
would they know what the previous location is? that may be a separate usability issue than this language, but just something to think about


maybe:

`Move this image back to its previous location before it was quarantined`

Reply:
The original location is stored in the item's metadata as a folder ID.  We could do a REST request and look it up -- but I think that is a feature that I'd rather see moved upstream to HistomicsUI and left for a later time.